gef.py

```diff
@@ -3393,41 +3393,40 @@ def regchanged_handler(event) -> None:
     return
 
 
-def load_libc_args() -> None:
+def load_libc_args() -> bool:
+    """Load the LIBC function arguments. Returns `True` on success, `False` or an Exception otherwise."""
```

Comment:
We could make this always boolean (no exception) by doing `if not gef.config.get("context.libc_args", False)`